### DISCUSSION TITLE
Automated E-Mail Meeting Invites.

### DISCUSSION THREAD

**Discussion by ghost** [Budibase/budibase#7980] [Help] [answered]
Hi all,

I hope everyone's well?

I am very new to the Budibase toolkit and was wondering if someone could point me in the direction of some documentation/previous user experiences on how to use the automation feature to distribute a meeting invite, preferably including having the meeting dates selected based on user input.

For context, I have configured the SMTP services and have a working automation to send a basic e-mail message that includes {{Trigger.lines}} taken from my user inputs.  

Thanks!

**Comment by melohagan** [comment]
Hey @Macademus 

Which platform are you looking to create your meeting invite on, e.g. Gmail? I don't think it'll be a standard process between the various platforms!

**Comment by ghost** [comment]
Hi @melohagan 

Preferably an outlook calendar invite would work best. I'm happy to learn/read if there's any documentation on this. 

Thank you for your reply 👍!

**Reply by melohagan** [reply]
Thanks! I'll have to look into it myself, but I'll see what I can do!

**Comment by melohagan** [comment, marked as answer]
Hey @Macademus 

I had a quick look at this, and there is more functionality needed on the Budibase end to get this to work. I've raised a feature request here: https://github.com/Budibase/budibase/issues/8198

**Reply by ghost** [reply]
I'll monitor this feature request.

Thank you for your continued efforts.
